src/backend/stream.rs

```diff
@@ -221,7 +243,23 @@ impl<'ctx> PulseStream<'ctx> {
                 return;
             }
 
-            if stm.input_stream.is_none() {
+            if stm.input_stream.is_some() {
+                cubeb_logv!("duplex output callback. input frame count");
```

Comment:
This log message seems incomplete?

Reply:
Yes thanks. It looks like this ringbuffer does not have a way to get the number of elements inside it.